doc/getting-started/getting-started.rst

```diff
@@ -47,8 +47,9 @@ Before you begin, make sure your machines have the following prerequisites:
 * Hardware specifications
 
   - Target board (see :ref:`hardware_tested`)
-  - Ubuntu 18.04 Desktop bootable USB disk: download the `Ubuntu 18.04.05 Desktop
-    ISO image <https://releases.ubuntu.com/18.04.5/>`_ and follow the `Ubuntu documentation
+  - Ubuntu 18.04 Desktop bootable USB disk: download the `Ubuntu 18.04.05
+    Desktop ISO image <https://releases.ubuntu.com/18.04.5/>`_ and follow the
+    `Ubuntu documentation
     <https://ubuntu.com/tutorials/create-a-usb-stick-on-ubuntu#1-overview>`__
     for instructions for creating the USB disk.
```

Comment:
"...follow the ...documentation... for creating the USB disk." (Delete "instructions for".)